krabs/krabs.cpp

```diff
@@ -4,12 +4,13 @@
 #include "krabs.hpp"
 
 // Note: This file exists to define the schema_locator
-// cache static variable. Due to issues with C++14 magic
+// cache static variables. Due to issues with C++14 magic
 // static and C++/CLI, we're unable to rely solely on
 // using magic static.
 
 // DO NOT ADD THINGS TO THIS FILE
 
 namespace krabs {
-    schema_locator schema_locator::singleton_;
+    DWORD schema_locator::index_;
```

Comment:
I know it's actually zero-init, but I don't like seeing "uninitialized" variables getting read.
```suggestion
    DWORD schema_locator::index_ = 0;
```